recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "tkwant" %}
-{% set version = "1.0.1" %}
+{% set version = "1.1.0a0" %}
```

Comment:
This would require complying with https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
good point, i change the naming accordingly.